src/pages/documentation/adobe-target/release-notes.md

```diff
@@ -6,6 +6,10 @@
 
 * Major version update for [Adobe Target](./index.md) for Adobe Experience Platform Mobile SDKs on Android compatible with Mobile Core 2.0.0. This library is now available as an [open source project on GitHub](https://github.com/adobe/aepsdk-target-android).
 
+Please note that the following APIs have been renamed for alignment with the Adobe Target Mobile SDK for iOS:
```

Comment:
Can you also add a note for the public classes that were moved into target subpackage?

Reply:
Thanks for the reminder 👍 will do